src/main/java/com/google/sps/data/PersistentUserRepository.java

```diff
@@ -176,24 +175,18 @@ public PreparedQuery getQueryFilterForId(String id) {
   }
   
   // function that fetches a single user, collection of users, filter on that user for id
-  public User fetchUserWithId(String userId) throws Exception{
+  public User fetchUserWithId(String userId) {
     PreparedQuery results = getQueryFilterForId(userId);
     Collection<User> userProfiles = fetchUserEntities(results);
     if(userProfiles.size() == 1){
```

Comment:
Can you think of any way to simplify the control flow here? It looks like there are three return statements for handling two cases.

Reply:
I simplified it, I removed the extra if, so theres return null if size isnt 1

Reply:
Done